src/common/io_exerciser/Model.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include "IoOp.h"
+
+#include <boost/asio/io_context.hpp>
+
+#include "librados/librados_asio.h"
+
+#include "include/interval_set.h"
+#include "global/global_init.h"
+#include "global/global_context.h"
+#include "common/Thread.h"
+
+/* Overview
+ *
+ * class Model
+ *   Virtual class. Models apply IoOps generated by an
+ *   IoSequence, they can choose how many I/Os to execute in
+ *   parallel and scale up the size of I/Os by the blocksize
+ *
+ */
+
+namespace ceph {
+  namespace io_exerciser {
+
+    class Model
+    {
+    protected:
+      int num_io;
+      const std::string oid;
```

Comment:
consider not using 'const' members, as this prevents moving.

Reply:
Was not aware of this nuance. Will update.